src/lib/Occurrence.hs

```diff
@@ -637,14 +671,46 @@ instance ApproxConst CollapsedUse where
 instance ApproxConst a => ApproxConst (Subset a) where
   approxConst (SAll use) = approxConst use
   approxConst (Inj (Just all_use) uses) =
-    -- TODO Short-circuit if the all_use is heavy enough
+    -- TODO(optimization) Short-circuit if the all_use is heavy enough
     approxConst all_use `plus` approxConst uses
   approxConst (Inj Nothing uses) = approxConst uses
   approxConst (Prod _ use) = approxConst use
 
 instance ApproxConst v => ApproxConst (M.IntMap v) where
   approxConst uses = foldl' max zero $ map approxConst $ M.elems uses
 
+-- === Usage info including static usage ===
+
+-- The `Int` is the count of static occurrences
+data AccessInfo n = AccessInfo Int (Access n)
```

Comment:
What are "static occurrences" and how do they differ from `Access`?

Reply:
Occurrences in the code, relevant for code size.  Brief comment added.